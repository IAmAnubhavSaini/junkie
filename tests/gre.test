#!/bin/sh

NAME="GRE"
PCAPS="pcap/gre/sample.pcap"
filter() {
	grep '^GRE' | cut -d' ' -f 2-
}
OUTPUT="gre.out"

. ./dump_test
