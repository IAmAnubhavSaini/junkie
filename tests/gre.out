-- pcap/gre/sample.pcap --
head_len=4, payload=84, version=0, protocol=2048, key=None
head_len=4, payload=84, version=0, protocol=2048, key=None
head_len=4, payload=84, version=0, protocol=2048, key=None
head_len=4, payload=84, version=0, protocol=2048, key=None
head_len=4, payload=84, version=0, protocol=2048, key=None
head_len=4, payload=84, version=0, protocol=2048, key=None
head_len=4, payload=84, version=0, protocol=2048, key=None
head_len=4, payload=84, version=0, protocol=2048, key=None
head_len=4, payload=76, version=0, protocol=2048, key=None
head_len=4, payload=76, version=0, protocol=2048, key=None
head_len=4, payload=60, version=0, protocol=2048, key=None
head_len=4, payload=60, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=75, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=76, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=540, version=0, protocol=2048, key=None
head_len=4, payload=540, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=76, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=476, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=468, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=788, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=52, version=0, protocol=2048, key=None
head_len=4, payload=61, version=0, protocol=2048, key=None
head_len=4, payload=89, version=0, protocol=2048, key=None
head_len=4, payload=61, version=0, protocol=2048, key=None
head_len=4, payload=89, version=0, protocol=2048, key=None
head_len=4, payload=76, version=0, protocol=2048, key=None
head_len=4, payload=76, version=0, protocol=2048, key=None
head_len=4, payload=76, version=0, protocol=2048, key=None
head_len=4, payload=76, version=0, protocol=2048, key=None
